documentation/modules/ROOT/pages/connectors/sqlserver.adoc

```diff
@@ -82,7 +82,7 @@ EXEC sys.sp_cdc_enable_table
 @source_name   = N'MyTable',
 @role_name     = N'MyRole',
 @filegroup_name = N'MyDB_CT',
-@supports_net_changes = 1
+@supports_net_changes = 0
```

Comment:
Based on what you say below, I don't think it should be the "default" shown here, as it's likely not what most people would want.

Reply:
Yes that's true,. The common use can would be to use net changes (only the most recent row for each PK). Here a recap on how net changes is implemented is SQl Server:
- keeping the CDC tables with "all changes", adding index on it to make the queries that keep oly the most recent row quick to execute
- and automatically creating a function that perform that querie.

However, Debezium does not use the net changes functions. It uses the all changes function (see below). So, enabling net change has no impact on the current working of Debezium except adding the load of maitaining an index that is never used (execpt if the user use the net change fuction outside of Debeziuml).

So, maybe a parameter should be added to Debezium to let the use chose which way to Query SQL Server ? Today it's defaulted to "all change" with parameter "all update old" has appears in this piece of code (SqlServerConnection.java > line 57):
`private static final String GET_ALL_CHANGES_FOR_TABLE = "SELECT * FROM cdc.[fn_cdc_get_all_changes_#](ISNULL(?,sys.fn_cdc_get_min_lsn('#')), ?, N'all update old')";`

Reply:
May be it can be let to 1. However I think we should keep the paragraphe explaining the way Debezium queryies SQL Server to give people all the element to let them configure SQL Server. In my use case I wanted the configuration that (supports Debezium) & (has the minimal impact on the Database)